lib/paper_trail/frameworks/rails/controller.rb

```diff
@@ -25,7 +25,7 @@ def self.included(controller)
       # @api public
       def user_for_paper_trail
         return unless defined?(current_user)
-        ActiveSupport::VERSION::MAJOR >= 4 ? current_user.try!(:id) : current_user.try(:id)
+        current_user.try!(:id)
```

Comment:
```suggestion
        current_user.try(:id) || current_user
```
What do you think of removing the rescuing?

Reply:
Remove the `rescue NoMethodError`? I like it. The "Commit Suggestion" feature is only showing a one-line change though, so can you please suggest the full multi-line change, or just push a commit to this branch?